recipes/hs-process/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "hs-process" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/bb/12/820c4894f45ebeb6338e9fa8c6f070aa5b869b97c4a1fefb04bd70f8417c/hs_process-0.0.3.tar.gz"
+  sha256: ec8adc1562e5d7175341f3a58a0e08fd3f72915fa6a2eba12c841aaf5397cf65
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Can this be noarch python?

Reply:
I don't know anything about noarch python. So maybe - where can I find out more or do some tests to find out?